builtins/amp-video.md

```diff
@@ -82,9 +83,10 @@ Similar to the `video` tag `controls` attribute - if present, the browser offers
 
 If present, will automatically loop the video back to the start upon reaching the end.
 
-**muted**
+**muted (deprecated)**
```

Comment:
Bunch of reasons:

- `muted` definitely make sense when combined with autoplay but we handle that automatically when `autoplay` is present. Essentially we are no longer supporting `unmuted autoplay` even on desktop as of this PR (`unmuted autoplay` is bad UX regardless of desktop or mobile) so when there is `autoplay`, there is no longer a need for `muted` attribute.

- building consistent expectation on what happens when user taps a video (regardless of autoplay). It should either always unmute or never unmute (Can't really see a use-case where a publisher can justifiably break that expectation for the user. User can always mute/change volume themselves)

- most importantly: in iOS, there is no unmute button! if a video is marked as `muted`, user can not unmute it (crazy). Do we want publishers to prevent a user from ever unmuting a video?

I am not super opposed to keeping this attribute, but I rather try to do this for the sake of better UX and if there is a backlash or reasonable use-cases for it, then bring it back again.

/cc @rudygalfi for thoughts.